orttraining/orttraining/core/framework/pipeline.h

```diff
@@ -322,13 +322,13 @@ struct PipelineTensorNames {
 
 struct PipelineContext {
   // Number of pipeline stages.
-  int num_pipeline_stages;
+  int num_pipeline_stages = 0;
```

Comment:
Give every variable a default value to make VC static analyzer happy. Also, I feel it's safer.